src/AzureFunctions/Properties.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0"?>
+<Content xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Object id="AZURE_FUNCTIONS">
+    <Group>
+      <Id>Cloud</Id>
+      <Name>Cloud Settings</Name>
+      <Description>Settings for Azure Functions deployment</Description>
+      <Type>Main</Type>
+      <Childs>
+	      <Prop>
+          <Id>AZURE_FUNCTIONS_FUNCTION_APP</Id>
+          <Name>Function app</Name>
+          <Description>Name of the Function app in Azure. It must start with a letter and only contain letters, numbers, _, and -, up to 127 characters in length.</Description>
+          <Type>Text</Type>
+          <Default></Default>
+      </Prop>
+	    <Prop>
+          <Id>AZURE_FUNCTIONS_RESOURCE_GROUP</Id>
+          <Name>Resource group</Name>
+          <Description>Name of the resource group where the function app is defined in Azure.</Description>
+          <Type>Text</Type>
```

Comment:
I think this Property is missing option type required.

Option required seems unimplemented in IDE, however, it's better to have the attribute here.